.github/workflows/build-wheels-python-dependent.yml

```diff
@@ -41,6 +41,13 @@ jobs:
             os: ubuntu-24.04-arm
           - python-version: ${{ inputs.oldest_supported_python }}
             os: linux-armv7-self-hosted
+          # Exclude Python 3.14 for linux-armv7-self-hosted (Docker image python:3.14-bullseye doesn't exist yet)
```

Comment:
Maybe they won't release it since it is almost at EOL. Can't you switch to bookworm?